### PR TITLE
refactor: flip assert and actual value position on tests

### PR DESCRIPTION
**Pull request by samsonasik**
**Description**

the actual value should be on the right.

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

**Comment by samsonasik** [comment]
Ready to merge 👍

File: tests/system/Router/RouterTest.php [hunk 1]
```diff
@@ -212,10 +212,10 @@ public function testClosures(): void
 
         $closure = $router->controllerName();
 
-        $expects = $closure(...$router->params());
+        $actual = $closure(...$router->params());
 
         $this->assertIsCallable($router->controllerName());
-        $this->assertSame($expects, '123-alpha');
+        $this->assertSame('123-alpha', $actual);
```
Comment (kenjis) [inline tests/system/Router/RouterTest.php:218]: Oh, Rector did this?

**Review by kenjis — APPROVED** [review]

File: tests/system/Router/RouterTest.php (hunk 1 above)
Comment (samsonasik) [reply tests/system/Router/RouterTest.php:218]: latest `main` branch yes 👍

**Comment by samsonasik** [comment]
Thank you @kenjis for the review 👍 , let's merge it ;)

**Pull request merged**